config/workload_all.xml

```diff
@@ -50,7 +50,7 @@
       Set the number of retries to 0 as retrying when the number of warehouses is
       high is pointless as it just leads to more failures.
     -->
-    <maxRetriesPerTransaction>2</maxRetriesPerTransaction>
+    <maxRetriesPerTransaction>3</maxRetriesPerTransaction>

```

Comment:
Please make it 2. Otherwise all the reports will be start showing 3 retries.

Reply:
Keeping the retries as 2